tests/test.linter.js

```diff
@@ -246,15 +246,15 @@ describe('Linter', () => {
     addonLinter.io = { files: { whatever: {} } };
     addonLinter.io.getFile = () => Promise.resolve();
     addonLinter.getScanner = sinon.stub();
-    class fakeScanner {
+    class FakeScanner {
```

Comment:
Thanks! (it seems a convention that it could be also nice to enforce on all our sources using an eslint rules, but we can evaluate that in a follow up because I guess that an eslintrc rule could catch more of these and they would be changes unrelated to this pull request goal).